cookiecutter/{{cookiecutter.organization}}-richie-site-factory/template/{{cookiecutter.site}}/src/frontend/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "richie",
+  "version": "1.18.0",
```

Comment:
```suggestion
  "version": "1.0.0",
```

Reply:
I propose ;`0.1.0` for their first version